engine/tester/config_parse.c

```diff
@@ -1087,7 +1184,27 @@ get_script(xmlNodePtr node, tester_cfg *cfg, test_script *script)
         break;
     }
 
-    if (!execute_found)
+    if (ritem->role == RI_ROLE_TEMPLATE)
+        return 0;
+
+    /* Copy values from template */
+    ADD_FROM_TMPL(objective);
```

Comment:
It does not look pretty with  if (tmpl_script != NULL) below.
It looks artificial when some copying is done outside of below check.
In fact, since macro is just 2 lines, I'd suggest to remove the macro completely. The code will look more consistent.